src/core/interface/submenu.sh

```
@@ -62,7 +51,8 @@ _submenu() {
 	fi
 
 	local help_message
-	help_message=$("$function_name" help 2>/dev/null || true)
+	help_message=$("$function_name" help "$parent_key" 2>/dev/null || true)
```

Shellcheck (0.10.0)

[warning] parent_key is referenced but not assigned.

(SC2154)